### ISSUE TITLE
Unable to access any kind of menu after new update on linux

### ISSUE THREAD

**Issue by mrhedx** [Yellow-Dog-Man/Resonite-Issues#2547]
### Describe the bug?

im unable to access any kind of menu on linux after the update regardless of distro or proton version

### To Reproduce

login

### Expected behavior

to not do that

### Screenshots

![Screenshot_20240711_220216](https://github.com/user-attachments/assets/d1721db2-9df9-4421-9e03-c2e3b5563aca)
![Screenshot_20240711_215308](https://github.com/user-attachments/assets/c5dae1e9-8ec9-4f53-abc2-a86a85ff407f)
![Screenshot_20240711_220216](https://github.com/user-attachments/assets/4b060763-c6ef-4411-ad8b-05c727f8aed6)

### Resonite Version Number

2024.7.12.12

### What Platforms does this occur on?

Linux

### Log Files

[PENIS - 2024.7.12.12 - 2024-07-11 22_31_50.log](https://github.com/user-attachments/files/16186534/PENIS.-.2024.7.12.12.-.2024-07-11.22_31_50.log)

**Comment by Electronus** [comment]
im going to guess its related to #2537 If not a duplicate honestly. 

TL;DR try -ResetDash.

**Comment by stiefeljackal** [comment]
> im going to guess its related to #2537 If not a duplicate honestly.
> 
> TL;DR try -ResetDash.

It's a duplicate. For now, add both `-ResetDash` and `-NeverSaveDash`; the last part is important if you have modified your Dash. If your Dash was never modified, then don't worry about `-NeverSaveDash`.

**Comment by Frooxius** [comment]
This is duplicate yes, closing this one.
